packages/libraries/core/react-native.config.js

```diff
@@ -1,3 +0,0 @@
-module.exports = {
-  reactNativePath: '../../node_modules/@office-iss/react-native-win32'
-};
```

Comment:
I also wanted to point out I deleted this... it feels weird to me a cross platform framework would require react-native-win32 as it's react native version